src/api.ts

```diff
@@ -24,4 +24,4 @@ function get<T>(url: string): Promise<T> {
 
 export const getTodos = () => get<Todo[]>('/todos');
 
-export const getUser = (userId: number) => get<User>(`/users/${userId}`);
+export const getUser = (userId: number | undefined) => get<User>(`/users/${userId}`);
```

Comment:
Never make requests that a guaranteed to fail. If userId is undefined, just don't call get user
```suggestion
export const getUser = (userId: number) => get<User>(`/users/${userId}`);
```